sdk/digitaltwins/dtdl-parser/src/parser/modelParserImpl.ts

```diff
@@ -208,7 +208,7 @@ export class ModelParserImpl implements ModelParser {
       let dependencyMapping: { [dtmi: string]: unknown } | null;
       try {
         dependencyMapping = await this.getModels(undefinedIdentifiers);
-      } catch (e: any) {
+      } catch (e) {
```

Comment:
Do we want this? Or is this just a prettier quirk using older typescript?

Reply:
It's the old v1 prettier that doesn't like `: any`.  This code was merged directly into main as the preparation to move to typescript v4.6 and now is failing the Check-Format step.

We could update prettier to v2 but that would requires re-formatting of everything.  So I chose to just fix the formatting expected by prettier v1.